helpers/date.js

```diff
@@ -0,0 +1,5 @@
+export const formatDate = date => {
+  const dateOptions = { weekday: 'long', month: 'long', day: 'numeric' };
+
+  return date.toLocaleString('en-EN', dateOptions).toUpperCase();
```

Comment:
Should we handle the upper-case from the `css` side? (i.e. via  a tailwind class) :) 

Also curious about what you think, I'm not sure 100% about it (IMO using css, we keep the information to the web browser about upper/lower cases, so Google Search results would display it without screaming... I think...)

Reply:
absolutely, let CSS handle it, done in 86c458775e4bb47df1b72df50f6dd54aba632aa2